### PR TITLE
enforce some object literal spacing rules

### PR DESCRIPTION
**Pull request by evnp**
http://eslint.org/docs/rules/key-spacing
http://eslint.org/docs/rules/object-curly-spacing

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -22,10 +22,12 @@ module.exports = {
     'eol-last':                    ['error'],
     'eqeqeq':                      ['error'],
     'indent':                      ['error', 2, {SwitchCase: 1}],
+    'key-spacing':                 ['error', {beforeColon: false, afterColon: true}],
```
Comment (nojvek) [inline index.js:25]: should we make this strict?

Comment (evnp) [reply index.js:25]: Personally think it's a bit draconian to prevent alignment of object values (like in this file for instance)

Comment (evnp) [reply index.js:25]: though actually you bring up the important point that `{mode: 'strict'}` is default, should be `minimum` if we want to allow that

Comment (nojvek) [reply index.js:25]: I guess the alignment makes sense. I see code like this in a number places. Breaking that wouldn't be so great.

```
      routes: {
        'report/:reportId':             this.routeHandlers.load,
        'report/:reportId/:reportName': this.routeHandlers.load,
        'reset':                        this.routeHandlers.reset,
        'learn':                        this.routeHandlers.learn,
        ':jsurl':                       this.routeHandlers.jsurl,
        '':                             this.routeHandlers.index,
      },
```

**Review by nojvek — APPROVED** [review]

**Comment by evnp** [comment]
Corresponding Insights change: https://github.com/mixpanel-platform/insights/pull/324

**Comment by nojvek** [comment]
Closing. Already part of 3.3.0

**Pull request closed without merging**